releasenotes/notes/logs-fix-udp-forwarder-without-lf-788bd5b4d88d0ba5.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed a bug where logs forwarded by UDP would not be split because of missing line feed character at the end of a datagram.
+    Now adding a line feed character at the end of each frame is deprecated because it is automacilly added by the agent on read operations.
```

Comment:
```suggestion
    Now adding a line feed character at the end of each frame is deprecated because it is automatically added by the agent on read operations.
```